.gitignore

```diff
@@ -2,3 +2,4 @@ __pycache__/
 build/
 *.egg-info/
 dist/
+pythonenv*/
```

Comment:
The `pythonenv*/` entry should exist in your [user-local](https://docs.github.com/en/free-pro-team@latest/github/using-git/ignoring-files#excluding-local-files-without-creating-a-gitignore-file) or [user-global](https://docs.github.com/en/free-pro-team@latest/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer) gitignore file.